autogen/retrieve_utils.py

```diff
@@ -335,12 +335,12 @@ def query_vector_db(
         and query function.
 
     Args:
-        query_texts (List[str]): the query texts.
+        query_texts (List[str]): the list of strings which will be used to query the vector db.
         n_results (Optional, int): the number of results to return. Default is 10.
         client (Optional, API): the chromadb compatible client. Default is None, a chromadb client will be used.
         db_path (Optional, str): the path to the vector db. Default is "/tmp/chromadb.db".
         collection_name (Optional, str): the name of the collection. Default is "all-my-documents".
-        search_string (Optional, str): the search string. Default is "".
+        search_string (Optional, str): the search string. Only docs containing this string will be retrieved. Default is "".
```

Comment:
Does "containing this string" mean exact match?

Reply:
Yes, it means exact match. Updated.